theatre/core/src/propTypes/internals.ts

```diff
@@ -17,6 +17,8 @@ export type IValidCompoundProps = {
   [K in string]: PropTypeConfig
 }
 
+// Question: Why doesn't this include Rgba? Should it include Marker?
+//  - Is this just for compound props?
```

Comment:
Because this line would become unpredictable:

```typescript
const obj = sheet.object('obj', {color: {r: 0, b: 0, g: 0, a: 0}})
```

* In Theatre 0.4.7, `obj.value.color` would be a compound prop. It cannot be sequenced, but its sub-props can.
* In Theatre 0.4.8, `obj.value.color` would be a simple prop. It _can_ be sequenced, while it sub-props _can't_.

Reply:
I've applied a comment to this effect, I think.